addons/web/static/tests/helpers/test_utils_mock.js

```diff
@@ -52,6 +52,9 @@ function _observe(widget) {
  */
 function removeSrcAttribute(el, widget) {
     var nodes;
+    if (el.nodeName === "#comment") {
+        return;
+    }
```

Comment:
what does this fix?

Reply:
template inheritance is adding a comment "<-- inherited form..." which is the element that is passed to that method, and it crashes because comments don't have "getElementsByTagName".
It doesn't make sense to try to "remove src attribute" from a comment tag.